src/openrct2/drawing/ScrollingText.cpp

```diff
@@ -1462,7 +1462,10 @@ int32_t scrolling_text_setup(
     // Setup scrolling text
     auto scrollText = &_drawScrollTextList[scrollIndex];
     scrollText->string_id = stringId;
-    std::memcpy(scrollText->string_args, gCommonFormatArgs, sizeof(scrollText->string_args));
+
+    auto ft = Formatter::Common();
+    ft.CopyInto<uint8_t*>(scrollText->string_args, sizeof(scrollText->string_args));
```

Comment:
scolling_text_setup should be modified to take a formatter making it not pass the values through the global variable. This will likely reduce the rendering issues that occur.

In a latter refactor all callers should be modified to create their own formatter not use the `Common` as this function is used multithreaded. Will require some further work to make this work so hence why it should be left for a later refactor.